index.js

```diff
@@ -362,6 +364,14 @@
         }
       });
 
+  //  keys :: StrMap a -> Array String
+  var keys = function(strMap) {
+    if (typeof Object.keys === 'function') return Object.keys(strMap);
```

Comment:
Now, what do I about the code coverage? None of the test environments we run against lack `Object.keys` but we really ought to test this branch of code. I tried just overwriting the `Object` prototype version of `keys` with `null` in a test but it caused all manner of other failures.